### ISSUE TITLE
Library load error in Xcode 6.4

### ISSUE THREAD

**Issue by Luavis** [daltoniam/SwiftHTTP#131]
I only add SwiftHTTP in Podfile and when I compile my workspace. It make library load error.
This is my Podfile content
```ruby
platform :osx, '10.9'

use_frameworks!

target 'Nyaa.swift' do

  pod 'SwiftHTTP', '0.9.5'

end

```
When I check loading library of SwiftHTTP with otool.

```
	@rpath/libswiftAppKit.dylib (compatibility version 0.0.0, current version 0.0.0)
	@rpath/libswiftCore.dylib (compatibility version 0.0.0, current version 0.0.0)
	@rpath/libswiftCoreGraphics.dylib (compatibility version 0.0.0, current version 0.0.0)
	@rpath/libswiftDarwin.dylib (compatibility version 0.0.0, current version 0.0.0)
	@rpath/libswiftDispatch.dylib (compatibility version 0.0.0, current version 0.0.0)
	@rpath/libswiftFoundation.dylib (compatibility version 0.0.0, current version 0.0.0)
	@rpath/libswiftObjectiveC.dylib (compatibility version 0.0.0, current version 0.0.0)
	@rpath/libswiftQuartzCore.dylib (compatibility version 0.0.0, current version 0.0.0)
	@rpath/libswiftSecurity.dylib (compatibility version 0.0.0, current version 0.0.0)
```
and Xcode error code is caused by loading swift runtime library.
```
dyld: Library not loaded: @rpath/libswiftAppKit.dylib
  Referenced from: /Users/Luavis/Projects/Nyaa.swift/build/Debug/SwiftHTTP.framework/Versions/A/SwiftHTTP
  Reason: image not found
```

Is there any solution for this problem?

p.s. It run in mac command line tool project.

**Comment by daltoniam** [comment]
I don't know if CocoaPods does this for you, but you will probably need to include the frameworks in the build.

https://github.com/daltoniam/SwiftHTTP#add-copy-frameworks-phase

**Comment by Luavis** [comment]
I've already done that. :)
This problem caused of swifthttp framework load libswiftCore and the others.
Becaues there is not any library in @rpath.
It makes library load error.

**Comment by daltoniam** [comment]
Interesting. Would you be able to provide a test/sample project that produces the error so I could troubleshoot it further?

**Comment by daltoniam** [comment]
Where you able to get this sorted out or create a test project to share?

**Comment by Luavis** [comment]
Sorry... I was really busy last one week.

I attach link of [repository](http://github.com/Luavis/nyaa.swift) that make error, and screenshot of log.
I also do my best what makes trouble

screenshot
<img width="749" alt="screen shot 2015-07-06 at 5 06 01 pm" src="https://cloud.githubusercontent.com/assets/1534596/8740788/8bd03ba2-2c8a-11e5-8493-2e2ad3948079.png">

**Comment by daltoniam** [comment]
I cloned the project down and took a look. This doesn't appear to be a SwiftHTTP issue, but more of a cocoaPods and a command line tool issue. I tried manually copying the framework to the /Library/Frameworks directory but just get the error above. SwiftHTTP doesn't make any calls to AppKit directory, so it make be something xCode does when creating the framework. The only solution I see is to copy the SwiftHTTP source directly into the project. This might just be a limitation of how Swift command lines apps work with third parties frameworks.
